<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheGameLoop</class>
 <widget class="QMainWindow" name="TheGameLoop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>theGameloop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="temps">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="lineWidth">
     <number>8</number>
    </property>
    <property name="text">
     <string>Temps: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTuto1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ color: white;}</string>
    </property>
    <property name="text">
     <string>Pour vous déplacer, utiliser les fléches gauche et droite</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTuto2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ color: white;}</string>
    </property>
    <property name="text">
     <string>ou les touches 'Q' et 'D'</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTuto4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ color: white;}</string>
    </property>
    <property name="text">
     <string>Appuyer sur espace pour sauter et pour lancer le jeu</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTuto3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sylfaen</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ color: white;}</string>
    </property>
    <property name="text">
     <string>Bonne chance !</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
